docs/src/overview.md

```diff
@@ -1,69 +1,66 @@
 ---
+title: Overview
 slug: /
 sidebar_position: 0
 ---
 
-# Overview
+Welcome to %%te%%'s documentation!
```

Comment:
Nice to have this ` %%te%%` tag. Thanks for adding it!

Reply:
I'm slowing introducing it...and had to switch from the initial `{{}}` syntax due to https://github.com/thin-edge/tedge-docs/pull/62.

Once everything is confirmed to be working correctly, we'll also add it into the doc example notes so people know how and when to use it.